vars/lvVITester.groovy

```diff
@@ -0,0 +1,5 @@
+def call(Test_path, lvVersion){
+   echo "Running VITester."
+   branchDirectory = WORKSPACE.substring(0, WORKSPACE.lastIndexOf("\\"))
+   labviewcli("-OperationName RunVITester -TestPath \"$WORKSPACE\\$Test_path\" -AdditionalOperationDirectory \"$branchDirectory\\niveristand-custom-device-testing-tools\\RunVITester\"", lvVersion)
```

Comment:
Add a `-LogFilePath` to somewhere in `$WORKSPACE` like what exists for other lvBuild scripts.

Reply:
Changed to 
```
def logFileName = getLogName(testPath)
labviewcli("-OperationName RunVITester -TestPath \"$WORKSPACE\\$testPath\" -AdditionalOperationDirectory \"$branchDirectory\\niveristand-custom-device-testing-tools\\RunVITester\" -LogFilePath \"$WORKSPACE\\lvVITester_${logFileName}.log\"", lvVersion)
```